implementations/rust/ockam/ockam_app_lib/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 //!
 //! It exposes C APIs that can be used by the frontend to interact with the application.
 //!
+#![recursion_limit = "256"]
```

Comment:
Why did you need to increase the limit?

Reply:
I keep getting a pretty ugly compilation error and that's what the compiler recommended to do to avoid the issue (basically increase that value)... It was caused by a third party library (aws-smithy I think). I was hoping that this would disappear after the recent deps updates we did, but it didn't.